parsing/pcoq.mli

```diff
@@ -269,3 +279,7 @@ type any_entry = AnyEntry : 'a Entry.t -> any_entry
 
 val register_grammars_by_name : string -> any_entry list -> unit
 val find_grammars_by_name : string -> any_entry list
+
+(** Parsing state handling in the STM *)
```

Comment:
Why is the STM mentioned here? AFAICT this module lives below it.

Reply:
Fixed.